test/fixtures/db_test_data.js

```diff
@@ -0,0 +1,88 @@
+module.exports = [{"status": "FAILED", "host": "doesnotexist.com", "updatedAt": {"timezone": "+00:00", "$reql_type$": "TIME", }, "path": "/", "id": "cc4238ab-69d1-4f46-a096-46279e1f0658", "createdAt": {"timezone": "+00:00", "$reql_type$": "TIME"}, "requestedUrl": "http://doesnotexist.com"},
```

Comment:
This looks like raw data straight out of rethinkdb as opposed to what the objects look like in JavaScript land.  The date records are incorrectly formatted. 

I noticed when running `npm test` the following error:

```Something went wrong during seeding: ReqlLogicError: Invalid time object constructed (no field `epoch_time`):```

Might be better to write a little node script that queries (using ReQL) the DB and pulls out the info you want into proper JSON.